DOC Improving the documentation for newcomers (contribution docs) (#1486)
fairlearn/fairlearn: docs/contributor_guide/contributing_documentation.rst

Documentation is formatted in restructured text (ReST) and the website is
built using `Sphinx <https://www.sphinx-doc.org/en/master/>`_ and some of its
extensions.

> Specifically, the website is available for a part of our previous releases to

Reviewer: ```suggestion
Specifically, the website is available for most previous releases to
```

The home page is static and can be found in a
`subfolder <https://github.com/fairlearn/fairlearn/tree/main/docs/static_landing_page>`_.

The `API reference <file:///Users/tamara/FOSS/fairlearn/docs/_build/html/api_reference/index.html>`_
documentation is also generated from the page structure defined
in the `docs folder <https://github.com/fairlearn/fairlearn/tree/main/docs/api_reference>`_.
However, the content of the generated pages is taken from the NumPy style docstrings
found in the files of the respective classes and methods. For example, take a look at the
`code and docstrings <https://github.com/fairlearn/fairlearn/blob/main/fairlearn/preprocessing/_correlation_remover.py>`_
of the :code:`CorrelationRemover` and the corresponding
`generated page <file:///Users/tamara/FOSS/fairlearn/docs/_build/html/api_reference/generated/fairlearn.preprocessing.CorrelationRemover.html>`_.

Contributing
^^^^^^^^^^^^

The instructions below are an addition to the instructions on the
:ref:`development_process` page. Please follow the installation
instructions on that page, and return to follow the further
documentation specific guidelines.

> You may also need to `install Pandoc <https://pandoc.org/installing.html>`_. and :code:`matplotlib`.

Reviewer: ```suggestion
You may also need to `install pandoc <https://pandoc.org/installing.html>`_. and :code:`matplotlib`.
```
